malsub/common/rw.py

```diff
@@ -5,30 +5,30 @@
 def validff(file):
     for f in file:
         try:
-            with open(f, mode='rb') as fd:
```

Comment:
I was using single quotes to differentiate single character strings or special "codes" like `rb` from regular double quoted strings.

Reply:
Understood. As you see, `black` supports heavy usage of double quotes anywhere possible. I will put `skip_string_normalization = true` value in black formatting so it won't enforce it, even though I do think it's better to stick to a single quote-style formatting